### PR TITLE
[testArmZabbixAPI] Add tests for "--load-old-events" option

### PR DESCRIPTION
**Pull request by ashie**
Related issues:

  * #915
  * #1058

### CODE REVIEW

**Comment by ashie** [comment]
I'll rebase it after the event-id-to-string branch is merged.

**Comment by kz0817** [comment]
>I'll rebase it after the event-id-to-string branch is merged.

I won't be long.

File: server/test/testArmZabbixAPI.cc [hunk 1]
```diff
@@ -817,6 +821,36 @@ void test_verifyEventsObtanedBySplitWay(void)
 		cppcut_assert_equal(true, *itr <= upperLimitOfEventsAtOneTime);
 }
 
+void test_getInitialEvent(void)
+{
+	ArmZabbixAPITestee armZbxApiTestee(setupServer());
+	armZbxApiTestee.testOpenSession();
+
+	bool lastOnly = true;
```
Comment (kz0817) [inline server/test/testArmZabbixAPI.cc]: Let's add 'const' for the variable that is never changed after its definition.

**Comment by ashie** [comment]
I close this issue & rebase it to run tests with changes in #1086.

**Pull request closed without merging**